src/Basement/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ set(SRC
 
 add_library(BasementOptions INTERFACE)
 if(NOT CMAKE_Fortran_COMPILER MATCHES "ifort")
-  target_compile_options(BasementOptions INTERFACE -ffloat-store)
+  target_compile_options(BasementOptions INTERFACE -ffloat-store -finit-local-zero)
```

Comment:
Geez what we go through to figure out what causes diffs. This is one of the slickest fixes I've seen in a while.

Reply:
Credit is due to @JasonGlazer for finding this compiler option.